yt_dlp/extractor/pbs.py

```diff
@@ -178,6 +178,7 @@ class PBSIE(InfoExtractor):
         (r'video\.wtjx\.org', 'WTJX Channel 12 (WTJX)'),  # http://www.wtjx.org/
         (r'video\.ideastations\.org', 'WCVE PBS (WCVE)'),  # http://ideastations.org/
         (r'video\.kbtc\.org', 'KBTC Public Television (KBTC)'),  # http://kbtc.org
+        (r'(www\.)?thirteen\.org', 'Thirteen (WNET'),  # https://thirteen.org
```

Comment:
make it a non-capturing group
```suggestion
        (r'(?:www\.)?thirteen\.org', 'Thirteen (WNET'),  # https://thirteen.org
```